api/services/control/control.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.36.9
+// 	protoc-gen-go v1.36.10
```

Comment:
Where is this version change coming from? I'd expect to see a change in go.mod related to this change or in a Dockerfile depending on where this version is specified.